src/lib/autoLauncher.ts

```diff
@@ -181,16 +184,15 @@ export class AutoLauncher {
       this.autoLauncher = linAL
     }
 
-    const config = appConfig.getAppConfig()
-    if (config) {
-      if (config.launchOnBoot)
-      {
-        // the app may be initialized before, but then user may have decided to move the app to another directory
-        // in this case, we have to delete the previous autoLaunch entry, and create a new one
-        await this.disable()
-        await this.enable()
-        this.enabled = (await this.isEnabled())
-      }
+    const config = await appConfig.read()
+    if (config.launchOnBoot)
+    {
+      // the app may be initialized before, but then user may have decided to move the app to another directory
+      // in this case, we have to delete the previous autoLaunch entry, and create a new one
+      // below disable is not creating console error, hence use it for this one
+      await this.disable()
+      await this.enable()
+      this.enabled = (await this.isEnabled())
```

Comment:
this part is a bit confusing:
Does not `this.enable()` update `this.enabled` property? (line 149)

Reply:
true, nice catch 👍